### ISSUE TITLE
Enclose conversational form within a DIV : Angular 6 

### ISSUE THREAD

**Issue by Vbboi24** [space10-community/conversational-form#285]
**Is your feature request related to a problem? Please describe.**
Yes, I would like to use this Awesome library in my Angular 6 Project but i could'nt find a documentation on that. Though [Angular Example](https://github.com/space10-community/conversational-form/issues/191#issuecomment-465125899) here shows how to use it in Angular. It covers how this library can be used with Angular. But the problem i have is : 
**I am not able to refer to a context in my DOM.It goes into "body" by default hiding all my sidebar content in my application. I am not able to restrict it to a DIV tag.**
**Describe the solution you'd like**
It will be great if there's a working example on the same on how the conversation can be encapsuled within a DIV tag.
It would be great to have an angular example with the context pointing to a DOM Element. so i can wrap it around within a component.I have tried the same here : https://stackblitz.com/edit/angular-conversational-form-mppzmd
**Additional context**
![image](https://user-images.githubusercontent.com/20842962/53323730-fb489080-3904-11e9-8a34-5f4cb2232c38.png)

**Comment by jenssogaard** [comment]
@Vbboi24 I see the issue. I'll try and sort this out.

**Comment by jenssogaard** [comment]
@Vbboi24 I have updated the Stackblitz example - I was appending the form and not the CF instance in the example. Have a look at line 35, before I appended `.formEl` it should just be `.el`. And then just make sure your container is `position: relative`, as in the example, as we append an `absolute` container.

[Link to the updated example](https://stackblitz.com/edit/angular-conversational-form-example)
